src/librustdoc/html/format.rs

```diff
@@ -788,14 +797,14 @@ fn fmt_type(t: &clean::Type, f: &mut fmt::Formatter, use_absolute: bool,
                 _ => {
```

Comment:
I think it would be better to add a case for `ResolvedPath` here to render the parenthesis rather than passing bool parameters everywhere.